canonical-websites/www.canonical.com
# add view for secure boot master certificate

## Done

* hosting binary file for secure boot certificate at /secure-boot-master-ca.crl

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8002/secure-boot-master-ca.crl](http://0.0.0.0:8002/secure-boot-master-ca.crl)
4. See that the binary file downloads

## Issue / Card

Fixes #298
